daemons/gptp/common/ptp_message.cpp

```diff
@@ -873,7 +875,7 @@ void PTPMessageFollowUp::sendPort(IEEE1588Port * port,
 	XPTPD_INFO("Follow-up Dump:\n");
 #ifdef DEBUG
 	for (int i = 0; i < messageLength; ++i) {
-		XPTPD_PRINTF("%d:%02x ", i, (unsigned char)buf[i]);
+		fprintf(stderr, "%d:%02x ", i, (unsigned char)buf_t[i]);
```

Comment:
No, i've changed it to printf to debug and forgot to change it back. Im keeping the XPTPD_PRINTF macro